lib/browser/objects-registry.js

```diff
@@ -20,6 +20,7 @@ class ObjectsRegistry {
   add(webContents, obj) {
     // Get or assign an ID to the object.
     let id = this.saveToStorage(obj);
+    this.storage[id].count++;
```

Comment:
Due to the cache of remote objects in renderer process, one object would only be referenced for once in one webContents, while `add` would be called for multiples. So this change will make some objects never get dereferenced.